openfecwebapp/templates/datatable.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'layouts/main.html' %}
+
+{% block title %}
+  Browse {{ title }}
+{% endblock %}
+
+{% block body %}
+
+<header class="page-header slab slab--primary">
+  <h1 class="page-header__title">{{ title }}</h1>
+</header>
+
+<section class="main__content--full">
+  <div class="data-container__widgets js-data-widgets">
+    <div class="js-filter-tags data-container__tags">
+      <div class="js-panel-controls panel__controls">
+        <button type="button" class="js-filter-toggle filters__toggle" aria-hidden="true">Add filters +</button>
+        <button type="button" class="js-filter-clear filters__toggle filters__clear" aria-hidden="true">Clear all filters</button>
+      </div>
+    </div>
+  </div>
+  {% include 'partials/' + slug + '-filter.html' %}
+  <div class="data-container">
+    <div id="{{slug}}" class="data-container__body fade-in">
+      {% include 'partials/tables/' + slug + '-table.html' %}
+    </div>
+  </div>
+</section>
+
+{% include 'partials/datatable-modal.html' %}
+
+{% endblock %}
+
+{% block scripts %}
+{% if office %}
+<script>
+var context = {
+  office: '{{ office }}'
+};
```

Comment:
Just adding a note that @noahmanger and I reviewed this together and all is fine for now.